fuzzing/engines/BUILD

```diff
@@ -68,6 +68,14 @@ java_fuzzing_engine(
     name = "jazzer",
     display_name = "Jazzer",
     launcher = "jazzer_launcher.sh",
-    library = "@jazzer//agent:jazzer_api_compile_only",
+    library = ":jazzer_stub",
     visibility = ["//visibility:public"],
 )
+
+java_library(
```

Comment:
Maybe explain in a comment the role of this rule? It seems to provide no functionality?

Reply:
I added a comment.